compiler/rustc_attr/src/builtin.rs

```diff
@@ -98,6 +98,15 @@ pub struct Stability {
     pub feature: Symbol,
 }
 
+impl Stability {
```

Comment:
I've added some helper methods to help with potential ambiguity for readers. This way it's immediately clear if it's regular stability or const stability being checked.